GuildCalculator/src/main/resources/templates/Home.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Home</title>
+</head>
+<body>
+You're hitting this page
+{{#recipe}}
+<ul>
+    <li>id: {{id}}</li>
+    <li>type: {{type}}</li>
+    <li>ingredients: {{ingredients}}</li>
```

Comment:
These two lines probably should be removed.  Then you can output the items and the output item later in the template.